llvm/tools/llvm-lsp/OptRunner.h

```diff
@@ -91,12 +91,12 @@ class OptRunner {
   llvm::Expected<SmallVector<std::pair<std::string, std::string>, 256>>
   getPassListAndDescription(const std::string PipelineText) {
     // First is Passname, Second is Pass Description.
-    auto MaybeOutErr = runShellOpt(
-        {"--print-pass-numbers", "--passes", PipelineText, "--disable-output"});
+    auto MaybeOutErr =
+        runShellOpt({"-S", "--print-pass-numbers", "--passes", PipelineText});
     if (!MaybeOutErr)
       return MaybeOutErr.takeError();
     auto [_, Stderr] = *MaybeOutErr;
-    SmallString<256> StderrContent;
+    SmallString<1024> StderrContent;
```

Comment:
It is more straightforward to use std::string here as this certainly not going to fit into 1024, or any reasonable small character limit.